bundle/manifests/oran-o2ims.clusterserviceversion.yaml

```diff
@@ -110,9 +110,32 @@ spec:
         name: ""
         version: v1
       specDescriptors:
-      - description: InputDataSchema encapsulates all the schemas required for ClusterTemplate.
-        displayName: Input Data Schema
-        path: inputDataSchema
+      - description: Characteristics defines a List of key/value pairs describing
+          characteristics associated with the template.
+        displayName: Characteristics
+        path: characteristics
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: Description defines a Human readable description of the Template.
+        displayName: Description
+        path: description
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: Metadata  defines a List of key/value pairs describing metadata
+          associated with the template.
+        displayName: Characteristics
+        path: metadata
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: Name defines a Human readable name of the Template.
+        displayName: Name
+        path: name
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: TemplateID defines a Identifier for the O-Cloud Template. This
+          identifier is allocated by the O-Cloud.
+        displayName: TemplateID
```

Comment:
Just noticed that this is "ID" - I think the O-RAN spec uses "Id" for these things. Lets make sure we are using "Id" everywhere for consistency.

Reply:
Fixed